spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/TestConstants.java

```diff
@@ -491,4 +491,15 @@ private static <T> T[] copy(T... elements) {
               3L, 5L, 6L, ImmutableList.of(new StringStruct("6:str6", "5:str5", "3:str3"))),
           new NumStruct(
               3L, 1L, 4L, ImmutableList.of(new StringStruct("3:str3", "1:str1", "4:str4"))));
+
+  public static final List<NumStruct> numStructDatasetForUnion =
```

Comment:
Shouldn't this be `numStructDatasetForWindow`?

Reply:
my bad. will change it